typing/typetexp.ml

```diff
@@ -165,15 +170,15 @@ module TyVarEnv : sig
   (* evaluate with a locally extended set of univars *)
 
   val ttyp_poly_arg :
-    poly_univars -> (string * Parsetree.jkind_annotation option) list
+    poly_univars -> (string * Parsetree.jkind_annotation option * int) list
```

Comment:
All callsites of this function ignores the new `int` value, so I think it's better to not add this.